doc/source/data/saving-data.rst

```diff
@@ -0,0 +1,198 @@
+.. _saving-data:
+
+===========
+Saving data
+===========
+
+Writing data to files
+=====================
+
+Writing data to local disk
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To save your :class:`~ray.data.dataset.Dataset` to local disk, call a method
+like :meth:`Dataset.write_parquet <ray.data.Dataset.write_parquet>`  and specify a local
+directory with the `local://` scheme.
+
+.. warning::
+
+    If your cluster contains multiple nodes and you don't use `local://`, Ray Data
+    writes different partitions of data to different nodes.
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.read_csv("example://iris.csv")
+
+    ds.write_parquet("local:///tmp/iris/")
+
+To write data to formats other than Parquet, read the
+:ref:`Input/Output reference <input-output>`.
+
+Writing data to cloud storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To save your :class:`~ray.data.dataset.Dataset` to cloud storage, authenticate all nodes
+with your cloud service provider. Then, call a method like
+:meth:`Dataset.write_parquet <ray.data.Dataset.write_parquet>` and specify a bucket or
+folder URI.
```

Comment:
what other cloud providers are supported?

Reply:
Just Azure, right? I've added a tab for Azure